contractcourt/utxonursery.go

```diff
@@ -284,15 +284,13 @@ func (u *UtxoNursery) Start() error {
 	// point forward, we must close the nursery's quit channel if we detect
 	// any failures during startup to ensure they terminate.
 	if err := u.reloadPreschool(); err != nil {
-		close(u.quit)
-		return err
+		log.Errorf("UtxoNursery unable to reload preschool: %v", err)
```

Comment:
This seems more like a bandaid than a good fix.  I think this will enable bad half-initialized states in the UTXO nursery (e.g., `reloadPreschool` could fail halfway through and the nursery would quietly forget about any UTXOs that hadn't been processed yet).

I think what we really want is to detect the "already confirmed" case and update the nursery to the correct state.

Reply:
IIRC the UtxoNursery has been deprecated, so there is a question of how defensive we need to be here. My main question is that assuming we discovered this by accident, how did this manifest in the first place? Did we run into this? If so, I'm inclined to agree with @morehouse in that we probably can't just quietly forget unless we can justify to ourselves why this won't create an irrecoverable state.

Reply:
I agree that this is a bandaid. @ProofOfKeags the utxonursery is still used for non-anchor channels so I think we still need to be defensive here. Would be better to understand the root cause of the issue before proceeding w/ this pr